cmd/diff.go

```diff
@@ -89,7 +89,7 @@ func (d *Diff) Run(cmd *cobra.Command, args []string) error {
 	if err == nil {
 		for i := range min(int(versionCount), len(vers)) {
 			if !firstOutput {
-				err := core.WriteNewlines(writer, config.Newlines.AfterChangelogVersion)
+				err := core.WriteNewlines(writer, cfg.Changelog.Newlines.After)
```

Comment:
This naming feels out of place should we reuse

```suggestion
				err := core.WriteNewlines(writer, cfg.ReleaseNotes.Section.Newlines.After)
```